src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -303,10 +306,11 @@ public void canOverrideRenewalWhenDateFallsOutsideOfTheDateRangesInTheFixedLoanP
     UUID dueDateLimitedPolicyId = loanPoliciesFixture.create(currentDueDateRollingPolicy)
       .getId();
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(currentDueDateRollingPolicy)` and remove `UUID dueDateLimitedPolicyId`?

Reply:
Done.